miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -477,15 +478,16 @@ end
 #!
 #! Output: []
 #!
-#! - i is the index of the consumed note.
-#! - CN1_SN is the serial number of consumed note 1.
-#! - CN1_SR is the script root of consumed note 1.
-#! - CN1_IR is the inputs root of consumed note 1.
-#! - CN1_VR is the vault root of consumed note 1.
-#! - CN1_NA is the number of assets in consumed note 1.
-#! - CN1_A1 is the first asset of consumed note 1.
-#! - CN1_A2 is the second asset of consumed note 1.
-proc.process_consumed_note
+#! Where:
+#! - i is the index of the input note.
+#! - CN1_SN is the serial number of input note 1.
```

Comment:
Pretty non-important because it's still explicit, but `CN1_XX` probably stood for `ConsumedNote1_XX` which decreasingly makes sense since the `ConsumedNote` term is disappearing so we might want to change that at some point.

Reply:
Agreed - I'm holding off on doing a bigger renaming pass until later. But yes, these will have to be renamed accordingly.